lib/store/actions.js

```diff
@@ -119,6 +121,41 @@ const actions = {
       subscription,
     };
   },
+
+  startWatchingLocation() {
+    return (dispatch) => {
+      dispatch(locationUpdateStarted());
+      watchID = navigator.geolocation.watchPosition((pos) => {
+        dispatch(locationChanged(pos));
```

Comment:
Can we be explicit and only pass lat/lon as that's all we're using right now. If we need other properties at a later date, we can expose them.